topbeat/docs/gettingstarted.asciidoc

```diff
@@ -107,13 +107,13 @@ By default, it's set to 10 seconds.
 
 * The `procs` option defines a list of regular expressions to match all the processes that need to be monitored. By default, all the running processes are monitored.
 +
-If you are not interested in monitoring processes, you can use:
+If you are not interested in collecting per-process statistics, you can use:
 +
 [source, shell]
 -------------------------------------
 input:
-  period: 10
-  procs: ["^$"]
+  stats:
+    process: false
```

Comment:
@monicasarbu is this `proc` or `process`? See https://github.com/elastic/beats/blob/1.1/topbeat/etc/topbeat.yml#L18